tools/report.sh

```diff
@@ -31,7 +31,7 @@ error() {
 if [[ -z ${BASH_VERSINFO+x} ]]; then
   error "No bash version information available, aborting"
 fi
-if [[ "$BASH_VERSINFO" -lt 4 ]]; then
+if [[ "${BASH_VERSINFO[*]}" -lt 4 ]]; then
```

Comment:
This is not correct, as we only care about the major version, so index 0. The same in the other places.

```sh
$ if [[ "${BASH_VERSINFO[*]}" -lt 4 ]]; then
  error "You need bash version >= 4 to run the script"
fi
bash: [[: 5 2 15 1 release x86_64-redhat-linux-gnu: syntax error in expression (error token is "2 15 1 release x86_64-redhat-linux-gnu")
```